lib/spack/docs/developer_guide.rst

```diff
@@ -407,15 +407,15 @@ the string that it detected to be the name and version. The
 print URLs that were not being parsed correctly.
 
 """"""""""""""""""
-``spack url test``
+``spack url summary``
```

Comment:
I renamed this command to `spack url summary`. As the person who developed it, the number of times I've accidentally ran `spack test url` instead is embarrassing.